### ISSUE TITLE
VOTING: I Will Not Follow the License if less than GPLv3+! 

### ISSUE THREAD

**Issue by antifa** [HelloZeroNet/ZeroNet#2276]
I warning everyone that I was who reported that various licenses in ZeroNet are terminated. For years nobody ever even think about it. 
I did upgraded the license to GPL-3.0-or-later in my repository and I will not change the license to any less than this.
The voting is a joke and I as a contributor and the one who reported that ZeroNet violated various licenses and that is even incompatible with some I demand to upgrade the license as seen in the pull request! 

@imachug and @krixano are trying very hard to make sure other license is used than GPL. imachug even claimed proprietary rights over his contributions and threatened me if I don't delete his code from ZeroNet. 

Everyone who votes must vote to GPLv3+ which will prevent anyone to claim proprietary rights! I need to open new issues because many thread locked for me the one who trying to fix ZeroNet! You trying to exclude me just as how you closed my pull request and stolen my code what is now part of ZeroNet core without even I got credit for it! 

Without me there will no voting at. I urge everyone to distribute ZeroNet under GPLv3+ to prevent anyone from harassing, threatening someone who contributing to ZeroNet and also to prevent anyone from suing anyone else who has a copy of ZeroNet in his repositories. 

I will not downgrade the license in my own repository no matter what! Better if everyone aware of that!

There is pull request already with the new license and unfortunately it is spammed by some people from 4chan yesterday. You can also find a few issues still open about the license.

I bet on it that again I will get no credit. @shortcutme while included my pull request in ZeroNet core he was closed the pull request in order to prevent me getting any credit as contributor. 

I did fixed the font issue previously in ZeroNet, MIME types, etc.. some of which is for years was a problem ZeroNet, and now I again fixing something, let's just call it `LICENSE`. 

I would say to every contributor except a very few who continuously attacking me: 

I'm extremely thankful for your contributions to ZeroNet and I hope now you will vote according to the best interest of the ZeroNet project not how others tell you to do so. 

GPLv3+ is in the best interest of ZeroNet! 

Thank you for your attention!

:pull-request: https://github.com/HelloZeroNet/ZeroNet/pull/2270 
:ref: https://github.com/HelloZeroNet/ZeroNet/issues/2273, https://github.com/HelloZeroNet/ZeroNet/issues/2265, https://github.com/HelloZeroNet/ZeroNet/issues/2268

**Comment by imachug** [comment]
> as a contributor

You haven't contributed a single line to ZeroNet afaik. Please show the stolen lines already!

> @imachug and @krixano are trying very hard to make sure other license is used than GPL.

Not really. As you see, most people have voted for GPLv3 so far. I guess that's what we'll have to use. 

> Everyone who votes must vote to GPLv3+ which will prevent anyone to claim proprietary rights

Nope, they should vote for GPLv3 without plus. This will allow us to choose whether only GPLv3 is applied or later versions count too.

> I will not downgrade the license in my own repository no matter what

Are you fine with a terminated (sic) license? Okay.

**Comment by imachug** [comment]
By the way, I'm not against including my changes to ZeroNet. I'm against including my code into your project under GPLv3+ license without asking me first.

**Comment by antifa** [comment]
I did not need to ask you because you was one who violated other peoples copyrights and licenses for years and this you never in your life can change it!

**Comment by antifa** [comment]
The voting is shady trash wrote by you and some another idiot. I will not follow any result considering the previous conversations in other issues. Everyone publicly can read your opinions there. You tried to revoke rights granted by GPL even after that I notified ZeroNet about serious licensing issues. Now trying to use shaddy voting which is not even mention a single reference to those conversations we had in the past 3 days. You are criminal copyright infingers and in some countries you can be locked up for 10 years for this! So now you try to blame me and attack me in various ways? I say shame on you!

**Comment by antifa** [comment]
ZeroNet is going to be on GPLv3+! **I refuse to comply with anything less than GPLv3+!** I already fixed the license in my repository and since **that is the only repository with the license which complies with all other licenses** and compatible with others, I request to merge my pull request to make ZeroNet GPLv3+ in this repository as well.

**Comment by imachug** [comment]
> I did not need to ask you because you was one who violated other peoples copyrights and licenses for years and this you, never in your life can change it!

Again: an invalid license doesn't make the whole project public domain! If you think so, just show me a line in some international law or GPL that states this.

**Comment by imachug** [comment]
> You tried to revoke rights granted by GPL even after that I notified ZeroNet about serious licensing issues

My code was licensed under GPLv2. You can't use it in a GPLv3+ project! GPLv2 doesn't allow you to do that.

**Comment by imachug** [comment]
> I already fixed the license in my repositor

No you didn't. You include my code that is GPLv2 and is thus incompatible with GPLv3+.

**Comment by antifa** [comment]
> > You tried to revoke rights granted by GPL even after that I notified ZeroNet about serious licensing issues
> 
> My code was licensed under GPLv2. You can't use it in a GPLv3+ project! GPLv2 doesn't allow you to do that.

You can't even use GPLv2 on ZeroNet legally! What are you talking about? 3 different licenses you violated for years!

**Comment by 0x6a73** [comment]
@HelloZeroNet I'm sorry that I have to ping you again, but it seems like trollls are opening threads on issues we've already solved or we're solving in an another issue. Thanks.

**Comment by 0x6a73** [comment]
@antifa As you've stated, the code is now in the public domain, which means that anyone can do whatever they want with it. Including creating a version of that code that's proprietary or non-GPL.

**Comment by antifa** [comment]
I opening the thread because I'm not able to vote or discuss elsewhere. The big anti-censors censoring me, my contributions etc. **The voting is a big trash!** Look at the wording of NON-EXISTENT LICENSES AND LICENSE COMBINATIONS!

They invented new licenses fast to confuse people!

**Comment by 0x6a73** [comment]
You're not able to discuss elsewhere because you're not discussing and not accepting the facts, what you're doing here is, as stated previously, constantly repeating the same thing. You didn't even answer my previous post and you're doing the thing that post was **exactly** addressing.

**Comment by antifa** [comment]
I'm sorry @0x6a73 I'm not against anybody here. 

But please look at this: 

GPLv3 and Lax if you accept switching to either GPLv3 or later or to **GPLv3-only** or a Lax/Permissive license
GPLv3+ and Lax if you accept switching to either GPLv3 or later or a Lax/Permissive license
GPLv3-only and Lax if you accept switching to either GPLv3-only or a Lax/Permissive license
GPLv3+ if you accept switching to GPLv3 or later
GPLv3-only if you accept switching to GPLv3-only
GPLv3 if you accept switching to either GPLv3 or later or to GPLv3-only

The above text copied from the "voting" is; 
2 times mentions: **+**, 
4 times mentions: **later**, 
6 times mentions: **only**

The voting should be like this: 
**GPL-3.0-only**
**GPL-3.0-or-later**
without additional misleading bull :hankey:

**Comment by 0x6a73** [comment]
Great. So you accept a legal possibility of Zeronet's code changing a license to something else than GPLv3+?

**Comment by antifa** [comment]
NO! I was who found the issue with the license and I fixed it accordingly, there is no need for voting and any result is unacceptable from such a fraud attempt to restrict the use of ZeroNet! 

**The voting is unethical and immoral because misleading those who had contributed to ZeroNet previously!**

**Comment by 0x6a73** [comment]
Fuck off then. You still didn't answer that question.

**Comment by 0x6a73** [comment]
As you've stated, the code is now in the public domain, which means that anyone can do whatever they want with it. Including creating a version of that code that's proprietary or non-GPL, right?

**Comment by antifa** [comment]
Choose your words you maniac! You spamming every single issue and pull request which exist here because you are a Polish neo-Nazi. Go and :gun: yourself.

**Comment by 0x6a73** [comment]
Whatever. Are you going to answer it or not?

**Comment by antifa** [comment]
I answer only to Allah!

**Comment by 0x6a73** [comment]
Wtf

**Comment by xddxd** [comment]
It all makes sense now lol

**Comment by antifa** [comment]
> Wtf

I said it right. You are a Polish neo-Nazi trash!

**Comment by 0x6a73** [comment]
@HelloZeroNet I think it's the right timeif you're able to do so) to ban @antifa from participating in these discussions altogether, since all he wants to do is to start flame wars, name people neo-nazis and say stuff completely irrelevant to these issues.

**Comment by antifa** [comment]
@0x6a73 You are the only one who keep posting off-topic! I can show you about 50 of your comments!

**Comment by 0x6a73** [comment]
Judging by how you were previously banned and your threads were locked, I don't think so. Also, telling someone "kill yourself" is a yet again illegal - encouraging suicides/violence.

**Comment by antifa** [comment]
I was a suggestion so is not illegal! :see_no_evil: :rofl:

**Comment by 0x6a73** [comment]
A suggestion means encouraging too. Whatever, I've already reported you to the Github Legal Team.

I'd like to remind you that Github is legally obligated to keep your data for the next 30 days for investigation purposes, so there'd be no problem if I reported you to the police just citing your current and your banned accounts.

I've taken screenshots of your posts just in case you delete them, but as I said, Github has to keep them in their database for 30 days exactly to counteract destroying evidence.

**Comment by antifa** [comment]
I wish to make an emoji which shows me making a big :hankey: pile on top of your head! 
(take a screenshot fast!)

**Comment by 0x6a73** [comment]
I don't care. I will stop replying to this thread until @HelloZeroNet undertakes any action regarding it.

**Comment by antifa** [comment]
Wow, I'm so scared! Please censor me! ZeroNet please censor me! No more free speech, no more anti-censorship come @shortcutme and ban me attack me come. Ohh yeah, the real face of zeroNet... Running by one person a a bunch of Nazi bots.
